tests/templates/kuttl/ldap-authentication/03_openldap.yaml.j2

```diff
@@ -44,7 +44,7 @@ spec:
             - name: LDAP_ADMIN_USERNAME
               value: admin
             - name: LDAP_ADMIN_PASSWORD
-              value: admin
+              value: "admin-pw withSpace$%\" &&} §"  # use spaces and special characters to make sure it doesn't break anything
```

Comment:
The password in `create_ldap_user.sh` should also be changed

Reply:
Actually, we should not bind from Druid with the admin password. I will update the PR to leave the admin password unchanged and instead bind with the integrationtest user (similar to https://github.com/stackabletech/nifi-operator/pull/627). Let's see how it goes